### ISSUE TITLE
Export button not working

### ISSUE THREAD

**Issue by rbrenesh** [mutovis/analysis-software#11]
Pressing on the export button returns the error:
```
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/batch_iv_analysis/gui.py", line 620, in handleButton
    self.exportInterp(row)
  File "/usr/lib/python3.7/site-packages/batch_iv_analysis/gui.py", line 533, in exportInterp
    a = np.transpose(a).astype(float)
ValueError: setting an array element with a sequence.

```

**Comment by greyltc** [comment]
Yep, I can recreate this here. Thanks!

**Comment by greyltc** [comment]
This should be fixed by e9cfeba in v3.0.7 I also added a few more columns for the raw device data in the exported file that weren't there previously.
